web-client/src/presenter/computeds/JudgeActivityReport/judgeActivityReportHelper.test.ts

```diff
@@ -0,0 +1,226 @@
+import {
+  CASE_STATUS_TYPES,
+  SESSION_TYPES,
+} from '../../../../../shared/src/business/entities/EntityConstants';
+import { applicationContextForClient as applicationContext } from '../../../../../shared/src/business/test/createTestApplicationContext';
+import { judgeActivityReportHelper as judgeActivityReportHelperComputed } from './judgeActivityReportHelper';
+import { judgeUser } from '../../../../../shared/src/test/mockUsers';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../../../withAppContext';
+
+describe('judgeActivityReportHelper', () => {
+  let mockJudgeActivityReport;
+  let mockForm;
+  let baseState;
+
+  const judgeActivityReportHelper = withAppContextDecorator(
+    judgeActivityReportHelperComputed,
+    { ...applicationContext },
+  );
+
+  beforeEach(() => {
+    mockJudgeActivityReport = {
+      casesClosedByJudge: {
+        [CASE_STATUS_TYPES.closed]: 1,
+        [CASE_STATUS_TYPES.closedDismissed]: 5,
+      },
+      opinions: [
+        {
+          count: 1,
+          documentType: 'Memorandum Opinion',
+          eventCode: 'MOP',
+        },
+        {
+          count: 0,
+          documentType: 'S Opinion',
+          eventCode: 'SOP',
+        },
+        {
+          count: 0,
+          documentType: 'TC Opinion',
+          eventCode: 'TCOP',
+        },
+        {
+          count: 4,
+          documentType: 'Bench Opinion',
+          eventCode: 'OST',
+        },
+      ],
+      orders: [
+        {
+          count: 1,
+          documentType: 'Order',
+          eventCode: 'O',
+        },
+        {
+          count: 5,
+          documentType: 'Order for Dismissal',
+          eventCode: 'ODS',
+        },
+      ],
+      trialSessions: {
+        [SESSION_TYPES.regular]: 1,
+        [SESSION_TYPES.hybrid]: 0.5,
+        [SESSION_TYPES.motionHearing]: 1.5,
+      },
+    };
+
+    mockForm = {
+      judgeName: judgeUser.name,
+    };
+
+    baseState = {
+      form: mockForm,
+      judgeActivityReportData: mockJudgeActivityReport,
+    };
+  });
+
+  describe('closedCasesTotal', () => {
+    it('should be the sum of the values of cases closed off state.judgeActivityReportData', () => {
+      const { closedCasesTotal } = runCompute(judgeActivityReportHelper, {
+        state: baseState,
+      });
+
+      expect(closedCasesTotal).toBe(6);
+    });
+  });
+
+  describe('isFormPristine', () => {
+    it('should be true when startDate is not populated', () => {
+      const { isFormPristine } = runCompute(judgeActivityReportHelper, {
+        state: {
+          ...baseState,
+          form: {
+            ...mockForm,
+            endDate: '01/02/2020',
+            startDate: undefined,
+          },
+        },
+      });
+
+      expect(isFormPristine).toBe(true);
+    });
+
+    it('should be true when endDate is not populated', () => {
+      const { isFormPristine } = runCompute(judgeActivityReportHelper, {
```

Comment:
I'm liking this `isFormPristine` idea and how it could be used as we march towards Continuous Deployment. 🤔 👏